cloudinit/distros/bsd.py

```diff
@@ -149,3 +150,23 @@ def get_proc_ppid(pid):
         """
         ppid, _ = subp.subp(["ps", "-oppid=", "-p", str(pid)])
         return int(ppid.strip())
+
+    @staticmethod
+    def get_mapped_device(blockdev: str) -> Optional[str]:
+        return None
+
+    @staticmethod
+    def device_part_info(devpath: str) -> tuple:
+        # FreeBSD doesn't know of sysfs so just get everything we need from
+        # the device, like /dev/vtbd0p2.
+        part = util.find_freebsd_part(devpath)
+        if part:
+            fpart = f"/dev/{part}"
+            # Handle both GPT partitions and MBR slices with partitions
+            m = re.search(
+                r"^(?P<dev>/dev/.+)[sp](?P<part_slice>\d+[a-z]*)$", fpart
+            )
+            if m:
+                return m["dev"], m["part_slice"]
+
+        return distros.Distro.device_part_info(devpath)
```

Comment:
I'm guessing that the lack of `super()` here is intentional due to it being a static method?